doc/api/cluster.md

```diff
@@ -425,19 +425,14 @@ added: v0.9.12
 * `signal` {string} Name of the kill signal to send to the worker
   process. **Default**: `'SIGTERM'`
 
-This function will kill the worker. In the primary, it does this
-by disconnecting the `worker.process`, and once disconnected, killing
-with `signal`. In the worker, it does it by disconnecting the channel,
-and then exiting with code `0`.
+This function will kill the worker. In the master, it does this by disconnecting
```

Comment:
Please keep `primary` and not use the word `master`.

Reply:
Fixed - thanks for noticing this :)